ang/test/civicase/activity/card/directives/activity-card.directive.spec.js

```diff
@@ -24,14 +25,66 @@
       });
     });
 
+    describe('when editing an activity in the popup', function () {
+      var activity, caseObj, loadFormSpy, crmUrlReturnVal, crmFormSuccessCallback;
+
+      beforeEach(function () {
+        crmUrlReturnVal = 'some string';
+        activity = activitiesMockData.get()[0];
+        caseObj = {
+          client: [{
+            contact_id: '1'
+          }]
+        };
+
+        activityCard.isolateScope().case = caseObj;
+
+        loadFormSpy = jasmine.createSpyObj('loadForm', ['on']);
+        loadFormSpy.on.and.callFake(function (event, fn) {
+          crmFormSuccessCallback = fn;
+        });
+
+        CRM.loadForm.and.returnValue(loadFormSpy);
+        CRM.url.and.returnValue(crmUrlReturnVal);
+
+        activityCard.isolateScope().viewInPopup(null, activity);
+      });
+
+      it('opens the modal to edit the activity', function () {
+        expect(CRM.url).toHaveBeenCalledWith('civicrm/case/activity', {
+          action: 'update',
+          id: activity.id,
+          cid: caseObj.client[0].contact_id,
+          caseid: activity.case_id,
+          reset: 1
+        });
+        expect(CRM.loadForm).toHaveBeenCalledWith(crmUrlReturnVal);
```

Comment:
Not sure if it's fine to use multiple expect inside a single it

Reply:
Fixed.